### ISSUE TITLE
Batch Normalization and Group Normalization | Seri Lee Blog

### ISSUE THREAD

**Issue by sally20921** [sally20921/sally20921.github.io#4]
https://sally20921.github.io/group.html 

 Batch normalization is used in most state-of-the-art computer vision techniques to stabilize training, but it also suffers from drawbacks. Group normalization can be an awesome alternative when the batch size is too small.
